lib/tanker/identity.rb

```diff
@@ -52,55 +44,61 @@ def self.serialize(hash)
 
     def self.create_identity(b64_app_id, b64_app_secret, user_id)
       assert_string_values({
-        app_id: b64_app_id,
-        app_secret: b64_app_secret,
-        user_id: user_id
-      })
+                             app_id: b64_app_id,
+                             app_secret: b64_app_secret,
+                             user_id: user_id
+                           })
 
       app_id = Base64.strict_decode64(b64_app_id)
       app_secret = Base64.strict_decode64(b64_app_secret)
 
-      raise ArgumentError.new("Invalid app_id") if app_id.bytesize != BLOCK_HASH_SIZE
-      raise ArgumentError.new("Invalid app_secret") if app_secret.bytesize != APP_SECRET_SIZE
-      raise ArgumentError.new("Invalid (app_id, app_secret) combination") if app_id != generate_app_id(app_secret)
+      raise ArgumentError, "Invalid app_id" if app_id.bytesize != Crypto::BLOCK_HASH_SIZE
+      raise ArgumentError, "Invalid app_secret" if app_secret.bytesize != APP_SECRET_SIZE
+      raise ArgumentError, "Invalid (app_id, app_secret) combination" if app_id != generate_app_id(app_secret)
 
-      hashed_user_id = hash_user_id(app_id, user_id)
+      hashed_user_id = Crypto.hash_user_id(app_id, user_id)
       signature_keypair = Crypto.generate_signature_keypair
       message = signature_keypair[:public_key] + hashed_user_id
       signature = Crypto.sign_detached(message, app_secret)
 
       serialize({
-        trustchain_id: Base64.strict_encode64(app_id),
-        target: 'user',
-        value: Base64.strict_encode64(hashed_user_id),
-        delegation_signature: Base64.strict_encode64(signature),
-        ephemeral_public_signature_key: Base64.strict_encode64(signature_keypair[:public_key]),
-        ephemeral_private_signature_key: Base64.strict_encode64(signature_keypair[:private_key]),
-        user_secret: Base64.strict_encode64(user_secret(hashed_user_id))
-      })
+                  trustchain_id: Base64.strict_encode64(app_id),
```

Comment:
same

Reply:
Same reason, this is the auto-formatter trying to align the arguments with the `{` opening brace =)